src/api/emojiReactionAdd.js

```diff
@@ -9,5 +10,6 @@ export default (
   apiPut(
     auth,
     `messages/${messageId}/emoji_reactions/${emojiName}`,
+    {},
```

Comment:
but its not a declaration its execution - is there any way in JS to pass a parameter with the name like:

```
getSomething("parameter1", paramter3="parameter3")
```